SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+IF YOU FIND A VULNERABILITY: Please create an issue. If you know how to resolve it, please create a PR.
```

Comment:
Add a email like privacy@thiojoe.com to report a security issue instead of creating a issue on the GITHUB.

Reply:
I changed it.